master/buildbot/statistics/storage_backends/influxdb_client.py

```diff
@@ -58,17 +47,20 @@ def __init__(self, url, port, user, password, db, captures,
                                      self.password, self.db)
         self.inited = True
```

Comment:
I agree with the proposed code.

Me suggesting `__something` was regarding "nothing is really private in python..." :smile: